scss/boost_union/post.scss

```diff
@@ -195,7 +195,17 @@ body.pagelayout-login:not(.loginbackgroundimage) #footnote {
     margin-bottom: 0;
 }
 
+/*=======================================
+ * Settings: Content -> Advertisement Tiles.
+ ======================================*/
 
+/* align-items-stretch d-flex align-self-stretch class in mustache template require to set width to inherit (Otherwise
+ width is flexible dependent on content). */
+
+.themeboostunionadvtileswrapper [class*="col-"] .card {
```

Comment:
Currently, the height of the card is only defined by its content. While this is not very likely, the admin _could_ configure a tile with just a background image and no content.
In this case, the tile is just 40-50 pixels high. It might make sense to add a `min-height: 175px;` to the card.